src/core/RecyclerListView.tsx

```diff
@@ -313,6 +318,10 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
         return viewabilityTracker ? viewabilityTracker.findFirstLogicallyVisibleIndex() : 0;
     }
 
+    public getLayoutDimension(): Dimension {
```

Comment:
For? Name of the method seems incorrect. Also, giving access to layouts open the array to mutation. There is an existing method for this already

Reply:
No this is different from getLayout. This just returns the scrollable height and width of the recycler view. Require it to manage sticky footers.

Reply:
Why do you need this?

Reply:
To manage sticky footers, I need to know when the scroll position + `RLV.height` is above or below a certain value. That's where I need this. Also setting `RLV.width` to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + `RLV.height` is above or below a certain value. That's where I need this. Also setting `RLV.width` to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + `RLV.height` is above or below a certain value. That's where I need this. Also setting `RLV.width` to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + RLV.height is above or below a certain value. That's where I need this. Also setting RLV.width to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + RLV.height is above or below a certain value. That's where I need this. Also setting RLV.width to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + RLV.height is above or below a certain value. That's where I need this. Also setting RLV.width to the width of the sticky list item.

Reply:
To manage sticky footers, I need to know when the scroll position + RLV.height is above or below a certain value. That's where I need this. Also setting RLV.width to the width of the sticky list item.